src/SkipOnEmptyCallback/SkipNever.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator\SkipOnEmptyCallback;
+
+final class SkipNever
```

Comment:
```suggestion
final class SkipNothing
```

Reply:
Decided to choose `SkipNone`, also it's a shorter.